CHANGELOG.md

```diff
@@ -8,10 +8,14 @@
 ### Changed
 
 * Deprecated `install_java_with_overlay` function. Buildpacks using this function should now use `install_openjdk` instead. See `README.md` for a usage example. ([#346](https://github.com/heroku/heroku-buildpack-jvm-common/pull/346))
+* `-XX:+UseContainerSupport` is no longer being set in `JAVA_TOOL_OPTIONS`. This flags is set by default since OpenJDK 10 and 8u191. All versions supported by this buildpack do have this default - therefore this change has no effect on the final JVM flags. ([#348](https://github.com/heroku/heroku-buildpack-jvm-common/pull/348))
```

Comment:
Should these go under `## [Unreleased]`?

Reply:
Thanks, this is probably because of my rebase! I'll fix the changelog (and the typos I just spotted too)